### PR TITLE
✨ <amp-ad> - Implement Seedtag adapter

### PR DESCRIPTION
**Pull request by ybootin**
Implement a seedtag ads adapter for `<amp-ad>` 

téléchargement
[seedtag](https://www.seedtag.com) is the The Leading Contextual Advertising Company

Can be tested with way : 
```
  <amp-ad
    width="320"
    height="250"
    type="seedtag"
    data-adunit-id="0"
    data-publisher-id="0000-0000-01"
    data-placement="inArticle"
    data-force-creative="https://creatives.seedtag.com/vast/ssp-responses/display-OM300x250.json">
  </amp-ad>
  ```

See ads/vendors/seedtag.md for more information

### CODE REVIEW

**Comment by alanorozco** [comment]
@ampproject/wg-monetization

File: 3p/integration.js [hunk 1]
```diff
@@ -523,6 +524,7 @@ register('rubicon', rubicon);
 register('runative', runative);
 register('sas', sas);
 register('seedingalliance', seedingalliance);
+register('seedtag', seedtag);
```
Comment (powerivq) [inline 3p/integration.js:527]: Change to this file is no longer needed. Please revert the change to `3p/integration.js`.

File: ads/vendors/seedtag.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/seedtag.js:2]: Nit: 2021

File: ads/vendors/seedtag.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/seedtag.md:2]: Nit: 2021
